README.md

```diff
@@ -42,9 +42,13 @@ generate this data from a wikidata dump.
 * $wgPropertySuggesterMinProbability - a float that sets a minimum threshold for suggestions (default 0.05)
 * $wgPropertySuggesterDeprecatedIds - a list of ints that blacklist suggestions
 * $wgPropertySuggesterInitialSuggestions - a list of ints that will be suggested when no statements exist
+* $wgPropertySuggesterClassifyingConditionWeight - a float between 0 and 1 that determines the weight that's given to classifying property ids relative to non classifying ones (default 0.5)
 
 ## Release notes
 
+### 4.0.0 (dev)
```

Comment:
I wonder why this additional setting requires such a major version bump? As far as I can tell this is not a breaking change.